coderedcms/api/mailchimp.py

```diff
@@ -40,17 +41,25 @@ def get_lists(self):
         return json_response
 
     def get_merge_fields_for_list(self, list_id):
-        endpoint = "lists/{0}/merge-fields?fields=merge_fields.tag,merge_fields.merge_id,merge_fields.name".format(list_id)
+        endpoint = '''
+        lists/{0}/merge-fields?fields=merge_fields.tag,
+        merge_fields.merge_id,
+        merge_fields.name
+        '''.format(list_id)
```

Comment:
Using triple string will preserve newlines, which will result in an invalid URL. In this case, you cannot split the string, just add noqa.